lib/common/formats.js

```diff
@@ -347,5 +347,69 @@ module.exports = {
       })
       .value();
     return JSON.stringify(to_ret, null, 2);
+  },
+
+  /**
+   * Creates a flatten JSON from input, capitalizing and then concatenating 
+   * each key with it's pre-key recursively.
+   *
+   * @memberof Formats
+   * @kind member
+   * @example
+   * ```json
+    {
```

Comment:
I think you need to have asterisks at the beginning of these lines because they need to be part of the comment block. Same for the other JSON examples in the comment below